internal/pkg/cli/job_deploy.go

```diff
@@ -168,9 +179,18 @@ func (o *deployJobOpts) Execute() error {
 			Tags:        tags.Merge(o.targetApp.Tags, o.resourceTags),
 		},
 		Options: deploy.Options{
-			ForceNewUpdate: o.forceNewUpdate,
```

Comment:
lol was this dead code? where `foreceNewUpdate` is always `false` because the flag doesn't exist?

Reply:
The true reason it is deemed "dead code" is because `forceNewUpdate` isn't used for `job` in `cli/deploy`.  Even when it's passed in as true, we don't force new update to the job.

In addition - yes the flag is not supported for `copilot job deploy`. But you could run `copilot deploy --forceNewUpdate` and the flag would be passed to job. Either way it took zero effect because it's not actually supported in `cli/deploy`.